### PR TITLE
SetLunchMaker Component completed

### PR DESCRIPTION
**Pull request by Maria-Chira**
SetLunchMaker Component Created, Update Login, Added Route Admin

## Related to
Trello Ticket:  Assigning Lunch Maker

- created the front-end for the SetLunchMaker Component;
- created a Modal Window to display all the lunch_makers assigned in the past
- added new Route for Admin Page
- created a new page for admin
- login component updated so the admin can be redirected on a different page.
- created 3 apis to respond to the front-end request
             1. get users by cohort id;
             2. update the is_lunch_maker value for the assigned lunch maker and add it to the lunch_maker_history table;
             3. get the history of all the lunch makers assigned in the past;

## Checklist:
- [x] My code follows the style guidelines of this project
- [x] I have carefully reviewed my own code
- [x] I have commented my code
- [x] I have updated any documentation

### CODE REVIEW

File: client/src/App.js [hunk 1]
```diff
@@ -4,13 +4,15 @@ import Student from "./pages/Student";
 import LoginPage from "./pages/LoginPage";
 import NavBar from "./components/NavBar";
 import Footer from "./components/Footer";
+import { Admin } from "./pages/Admin";
```
Comment (TomRafferty) [inline client/src/App.js:7]: do we need the selection brackets here? See line 4.

(!this won't cause any actual issues!)

**Review by TomRafferty — APPROVED** [review]
This all looks excellent!
Brilliant work as always :D keep it up!

File: client/src/App.js (hunk 1 above)
Comment (Maria-Chira) [reply client/src/App.js:7]: Thanks for pointing it out. I think the import was done automatically that's why we've got the brackets. It won't affect, but I've already removed them, the changes will appear in my next PR..

File: server/api.js [hunk 2]
```diff
@@ -88,6 +87,55 @@ router.get("/users/:id", (req,res)=> {
...
+			await trx("users").update("is_lunch_maker", true).where("id", lunchMakerId);
+
+			//inserted the nominated lunch_maker name and date into the lunch_maker_history table
+			await trx("lunch_maker_history").insert({ lunch_maker_name: lunchMakerName, created_on: "NOW()" });
+			res
+			.status(201)
+			.json({ msg: "The lunch maker was nominated successfully" });
+		});
+	} catch (error) {
+		res.status(401).json({ msg: "Something went wrong. Please try again later!" });
```
Comment (CanaryWharf) [inline server/api.js:134]: 401 specifically should only be used for Unauthorized errors. Which means you are unable to determine who the user is (Wrong password/ bad cookie/ etc)

**Review by Johng117 — APPROVED** [review]
LGTM, nice job Maria.

File: server/api.js (hunk 2 above)
Comment (Maria-Chira) [reply server/api.js:134]: Thank you Raj, I'll set the status to 500. :)

**Pull request merged**